tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -660,94 +606,71 @@ def _test_cpu_wrmsr_restore(context):
     )
 
     # Dump MSR state to a file for further comparison
-    msrs_after_fname = Path(snapshot_artifacts_dir) / shared_names["msrs_after_fname"]
+    msrs_after_fname = snapshot_artifacts_dir / shared_names["msrs_after_fname"]
     dump_msr_state_to_file(msrs_after_fname, vm.ssh, shared_names)
 
     # Compare the two lists of MSR values and assert they are equal
     check_msr_values_are_equal(
         Path(snapshot_artifacts_dir) / shared_names["msrs_before_fname"],
         Path(snapshot_artifacts_dir) / shared_names["msrs_after_fname"],
-        context.kernel.base_name(),  # this is to annotate the assertion output
+        guest_kernel.base_name(),  # this is to annotate the assertion output
```

Comment:
if this becomes an explicit input that will be visible in the report, can we drop it now?

Reply:
Yeah I can simplify that